File: src/TaskOTime/TaskOTime.ViewModel/ViewModels/TimeItemsViewModel.vb
Imports ActiveDevelop.TimeTrackingServices

Namespace ViewModels
    ''' <summary>
    '''  gebruikt de eigen sorterende collectie rechtstreeks voor de zichtbare tijdregels.
    ''' </summary>
    ''' <remarks>
    '''  deze klasse voegt geen tweede lijst toe.  de eigenschappen
    '''  <see cref="TimeCollectionViewModel.TimeItems"/> en
    '''  <see cref="TimeCollectionViewModel.SelectedDayEntries"/> verwijzen naar hetzelfde collectieobject.
    ''' </remarks>
    Public Class TimeItemsViewModel
        Inherits TimeItemsBase(Of Guid, TimeEntryViewModel)
    End Class
End Namespace

File: src/TaskOTime/TaskOTime.ViewModel/ViewModels/TimeItemsViewModel.cs
using System;
using ActiveDevelop.TimeTrackingServices;

namespace TaskOTime.ViewModel.ViewModels
{
    /// <summary>
    /// Exposes visible time entries directly through the custom timestamp-sorted collection.
    /// </summary>
    /// <remarks>
    /// This class does not maintain a second list. The
    /// <see cref="TimeCollectionViewModel.TimeItems"/> and
    /// <see cref="TimeCollectionViewModel.SelectedDayEntries"/> properties reference the same collection
    /// instance, including its ordering, neighbor links and change notifications.
    /// </remarks>
    public class TimeItemsViewModel : TimeItemsBase<Guid, TimeEntryViewModel>
    {
    }
}